containers/Connector/config.ts

```diff
@@ -51,26 +51,24 @@ export const initOnboard = (network: Network, subscriptions: Subscriptions) => {
 					appUrl: 'https://www.synthetix.io',
 					email: 'info@synthetix.io',
 					rpcUrl: infuraRpc,
+					preferred: true,
 				},
 				{
 					walletName: 'walletConnect',
 					rpc: { [network.id]: infuraRpc },
 					preferred: true,
 				},
-				{ walletName: 'coinbase', preferred: true },
+				{ walletName: 'imToken', rpcUrl: infuraRpc, preferred: true },
```

Comment:
I'm not seeing imToken wallet in our connect wallet model?
![image](https://user-images.githubusercontent.com/4015977/122136607-d0094880-ce10-11eb-9d8e-4ce81afc6354.png)

Reply:
@JChiaramonte7  yeah all good it's mobile only